client/jetbrains/src/main/java/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.java

```diff
@@ -84,6 +94,20 @@ public void clearAutocompleteSuggestions(@NotNull Editor editor) {
     disposeInlays(editor);
   }
 
+  /**
+   * Clears any already rendered autocomplete suggestions for all open editors and cancels any
+   * pending ones.
+   */
+  @RequiresEdt
+  public void clearAutocompleteSuggestions() {
```

Comment:
Nit: given it touches all projects, which is unusual, it might be good to emphasize in the method name
```suggestion
  public void clearAutocompleteSuggestionsForAllProjects() {
```